content/reference-capabilities/capability-matrix.md

```diff
@@ -0,0 +1,53 @@
+
+You'll notice that the top-right corner is empty. That's because, as previously discussed, we cannot make any local guarantees that are more restrictive than the global guarantees, or we'd end up with invalid capabilities that could be written to in this actor but read somewehre else at the same time.
+
+The matrix also helps visualizing other concepts previously discussed in this chapter:
+
+* __Sendable capabilities__. If we want to send references to a different actor, we must make sure that the global and local aliases make the same guarantees. It'd be unsafe to send a `trn` to another actor, since we could possibly hold `box` references locally. Only `iso`, `val`, and `tag` have the same global and local restrictions – all of which are in the main diagonal of the matrix.
+* __Ephemeral subtyping__. If we have an ephemeral capability (for instance, `iso^` after consuming an isolated variable), we can be more permissive for the new alias, i.e. remove restrictions, such as allowing local aliases with read capabilities, and receive the reference into a `trn^`; or both read and write, which gives us `ref`. The same is true for more global alias, and we can get `val`, `box`, or `tag`. Visually, this would be equivalent to walking downwards and/or to-the-right starting from the capability in the matrix.
+* __Recovering capabilities__. This is when we "lift" a capability, from a mutable reference to `iso` or an immutable reference to `val`. The matrix equivalent would be walking upwards starting from the capability – quite literally lifting in this case.
+* __Aliasing__. This is a bit less intuitive than the other points, since it only applies to `iso!` (which aliases to `tag`), and `trn!` (which aliases to `box`). If you picture a mirror in the anti-diagonal of the matrix (on top of `ref` and `val`), we can imagine that `iso` would be reflected on top of `tag` and `trn` would be reflected on top of `box`. Why? Well, our guarantees must be respected in new aliases to our references, . `iso` forbids reads/writes anywhere, so the new alias must prevent the programmer from doing any of these two actions locally and globally, which is why we end up with `tag`; and `trn` denies local and global writes (rendering it immutable), _but_ allows local reads, which gives us the locally immutable `box`, which is another way to say "it cannot be written to, at least from this reference". And since all other capabilities alias to themselves, we can simply follow the same rule for ephemeral subtyping: walking downwards and/or to-the-right.
```

Comment:
I'm not particularly fond of this item, since it doesn't actually give much visual intuition, and might drop it altogether.